rs-libc/src/alloc.rs

```diff
@@ -0,0 +1,76 @@
+#![allow(non_camel_case_types)]
+
+use std::ptr;
+use std::mem;
+use std::alloc::{GlobalAlloc, System, Layout};
+
+static A: System = System;
+
+// definition copied from std::os::raw::c_void
+#[repr(u8)]
+pub enum c_void {
```

Comment:
You can use `std::ffi::c_void` now. And remove the `#![allow(non_camel_case_types)]`